Python/navio/Navio2/Led.py

```diff
@@ -0,0 +1,38 @@
+ON = 0
+OFF = 1
+
+
+class Pin:
+    def __init__(self, folder_name):
+        self.pin = folder_name
+
+    def write(self, value):
+        with open("/sys/class/leds/%s/brightness" % self.pin, "w") as value_file:
+            value_file.write(str(value))
+
+
+class Led:
+    def __init__(self):
+        self.ledR = Pin("rgb_led0")
+        self.ledB = Pin("rgb_led1")
+        self.ledG = Pin("rgb_led2")
+
+        self.ledR.write(OFF)
+        self.ledG.write(OFF)
+        self.ledB.write(OFF)
+
+    def setColor(self, color):
```

Comment:
Please, bring the code to PEP8 compliance. I see a lot of camelCase naming here
```suggestion
    def set_color(self, color):
```